src/common/res/features/l10n/en.json

```diff
@@ -0,0 +1,423 @@
+    }
+  },
+  "Budget" : {
+    "Header" : {
+      "Totals" : {
+        "TBB" : "To be Budgeted",
+        "Funds" : "Funds for",
+        "Overspent" : "Overspent in",
+        "Budgeted" : "Budgeted in",
+        "BIF" : "Budgeted in Future"
+      },
+      "Metric" : {
+        "AoM" : "Age of Money",
+        "DoB" : "Days of Buffering"
+      }
+    },
+    "Table" : {
+      "Button" : {
+        "CategoryGroup" : "Category Group"
+      },
+      "Placeholder" : {
+        "NewCategoryGroup" : "New Category Group",
+        "NewCategory" : "New Category"
+      },
+      "Header" : {
+        "Category" : "CATEGORY",
+        "Budgeted" : "BUDGETED",
+        "Activity" : "ACTIVITY",
+        "Available" : "AVAILABLE",
+        "Pacing" : "PACING"
+      },
+      "Category" : {
+        "HiddenCategories" : "Hidden Categories"
+      }
+    },
+    "Inspector" : {
+      "Header" : {
+        "Budgeted" : "TOTAL BUDGETED",
+        "Activity" : "TOTAL ACTIVITY",
+        "Available" : "TOTAL AVAILABLE",
+        "Inflows" : "TOTAL INFLOWS",
+        "Quick" : "QUICK BUDGET",
+        "QuickSelected" : "QUICK BUDGET SELECTED",
+        "Payment" : "Payment",
+        "Goals" : "Goals",
+        "Notes" : "Notes",
+        "CategoriesSelected" : "Categories Selected"
+      },
+      "Title" : {
+        "CashLeft" : "Cash Left Over From",
+        "BudgetedThisMonth" : "Budgeted This Month",
+        "CashSpending" : "Cash Spending",
+        "CreditSpending" : "Credit Spending",
+        "Activity" : "Activity This Month",
+        "Available" : "Available",
+        "GoalTarget" : "Target Category Balance",
+        "GoalTargetByDate" : "Target Category Balance by Date",
+        "GoalMonthly" : "Monthly Funding Goal",
+        "Complete" : "Complete",
+        "Budgeted" : "Budgeted",
+        "ToGo" : "To go",
+        "TargetBalance" : "Target Balance",
+        "TargetBudgetedAmount" : "Target Budgeted Amount",
+        "TargetMonthYear" : "Target Month & Year"
+      },
+      "Button" : {
+        "Underfunded" : "Underfunded",
+        "BudgetedLastMonth" : "Budgeted Last Month",
+        "SpentLastMonth" : "Spent Last Month",
+        "AverageBudgeted" : "Average Budgeted",
+        "AverageSpent" : "Average Spent",
+        "CreateGoal" : "Create a goal",
+        "GoalTarget" : "Goal Target",
+        "UpcomingTransactions" : "Upcoming Transactions",
+        "BalanceToZero" : "Balance to 0.00:"
+      },
+      "Text" : {
+        "Message" : {
+          "HaventBudgeted" : "You haven't budgeted enough for the ",
+          "InUpcoming" : " in upcoming bills.",
+          "Overspent" : "You've overspent this category by ",
+          "Cover" : ". Cover the overspending from other categories or you can’t trust your budget balances!",
+          "YouRe" : "You're ",
+          "ShortOfGoal" : " short of your goal for the month. Budget another ",
+          "ToKeepTrack" : " to keep on track."
```

Comment:
There may not be a good way to do this at all with this library, but the ideal for things like "You're ____ short of your goal for the month. Budget another ______ to keep on track." would be to put a format string in there so if the order of the words needs to change in the translation, the translator can change it.